pxtblocks/fields/field_parameter.ts

```diff
@@ -0,0 +1,132 @@
+/// <reference path="../../localtypings/pxtblockly.d.ts" />
+
+declare namespace Blockly {
+    interface Workspace {
+        createVariable(name: string): VariableModel;
+        getVariable(name: string): VariableModel;
+        renameVariable(oldName: string, newName: string): void;
+    }
+}
+
+declare namespace Blockly.Msg {
+    const RENAME_VARIABLE_TITLE: string;
+    const VARIABLE_ALREADY_EXISTS_FOR_ANOTHER_TYPE: string;
+}
+
+declare namespace Blockly.Events {
+    const VAR_DELETE: string;
```

Comment:
localizable?

Reply:
Yep, these are both already localized in blocklyloader